crates/sui-config/src/node.rs

```diff
@@ -29,6 +29,12 @@ use sui_types::multiaddr::Multiaddr;
 // Default max number of concurrent requests served
 pub const DEFAULT_GRPC_CONCURRENCY_LIMIT: usize = 20000000000;
 
+/// Default gas price of 100 Mist
+pub const DEFAULT_VALIDATOR_GAS_PRICE: u64 = 100;
```

Comment:
@bmwill  any concerns with this (it was 10)

Reply:
Can we set this to 1000 instead?

Reply:
now it's 1000